django/contrib/humanize/templatetags/humanize.py

```diff
@@ -23,11 +23,17 @@ def ordinal(value):
         value = int(value)
     except (TypeError, ValueError):
         return value
-    suffixes = (_('th'), _('st'), _('nd'), _('rd'), _('th'), _('th'), _('th'), _('th'), _('th'), _('th'))
     if value % 100 in (11, 12, 13):  # special case
-        return mark_safe("%d%s" % (value, suffixes[0]))
+        return mark_safe(pgettext('ordinal 11, 12, 13', '{}th').format(value))
+    templates = (
+        pgettext('ordinal 0', '{}th'), pgettext('ordinal 1', '{}st'),
```

Comment:
Would it be a good idea to Add comments for translators (see https://docs.djangoproject.com/en/2.0/topics/i18n/translation/#comments-for-translators ) giving them hints about what these literal are? (This would mean putting each literal in its own line.)

Reply:
Cool, I didn’t know this exists. Some notes are added, please do tell me if I can make them better.